src/Features/Core/Portable/RemoveUnusedParametersAndValues/AbstractRemoveUnusedParametersAndValuesDiagnosticAnalyzer.SymbolStartAnalyzer.cs

```diff
@@ -183,13 +183,15 @@ private bool IsUnusedParameterCandidate(IParameterSymbol parameter)
                 // we don't have to bail out on reporting unused parameters for all public methods.
 
                 if (parameter.IsImplicitlyDeclared ||
+                    parameter.Name.Length == 0 ||
```

Comment:
Is it possible to check for error in this case and bail early?

Reply:
Could we also consider just not doing this analysis on methods that have *any* syntax errors in them.  I think it's a bit odd to think of something as "unused" when we dont' even know what the code means :)